### ISSUE TITLE
Can't pair Moonlight Steam Link with PC in v4.0.0

### ISSUE THREAD

**Issue by khiven** [moonlight-stream/moonlight-qt#772]
**Describe the bug**
I had a previous setup working. Came back after a year and updated the link to latest build and also installed latest Moonlight version. 
Now it will not pair with the PC through Moonlight, it will keep showing "Searching for PCs on your local network with NVIDIA GameStream enabled"

My PC has NVIDIA GFE installed and Game Stream enabled. My steam link is already paired with my steam pc and working fine.

I also tried a factory reset just in case something from previous builds was messing up, then updated steam link to latest build then installed Moonlight 4.0.0, same results.

**Steps to reproduce**
Just use the versions mentioned and try to pair a PC in the Moonlight Steam Link app.

**Screenshots**
N/A

**Affected games**
N/A 

**Other Moonlight clients**
N/A

**Moonlight settings (please complete the following information)**
Default settings

**Client PC details (please complete the following information)**
 - Steam Link
 - Moonlight Version: v4.0.0
 - Steam Link Build: 827

**Server PC details (please complete the following information)**
 - OS: Windows 11 Pro 10.0.22000 Build 22000
 - GeForce Experience version: 3.25.1.27
 - Nvidia GPU driver: 512.15
 - Antivirus and firewall software: Windows Defender

**Moonlight Logs (please attach)**
Cant get them from steam link (will do if you tell me how)

**Comment by cgutman** [comment]
Duplicate of https://github.com/moonlight-stream/nvidia-gamestream-issues/issues/30

**Comment by khiven** [comment]
My PC name nor my user have any non ASCII characters so I don't see how it is duplicated with https://github.com/moonlight-stream/nvidia-gamestream-issues/issues/30

Just wanted to give a hand but ok, I'll move on.

**Comment by cgutman** [comment]
Do you have your Steam Link configured for SSH? If so, you can pull the logs from Moonlight. They are in `/tmp/moonlight.log`. GFE's host-side logs are in `C:\ProgramData\NVIDIA Corporation\NvStreamSrv` (though the pairing logs are quite sparse on their side).

I do suspect this a GFE-side bug though, because Moonlight's pairing code hasn't changed in years.

**Comment by cgutman** [comment]
Oh I see, this doesn't look like a pairing issue at all, instead it looks like a connectivity or discoverability issue. You aren't even making it to the pairing process at all. Discovery and pairing still works for me on Steam Link with v4.0.0 (and v4.1.0), so I think it's an issue on your end.

Discovery issues are often caused by broken multicast support on your network or access point, which impede automatic discovery but connecting manually works.

Connectivity issues can be caused by settings like client isolation on your WiFi AP or when your host and client are connected to different networks (such as your home network and a guest network).

I suggest trying the Add PC button and typing your PC's IP address manually. Unfortunately, the Steam Link virtual keyboard doesn't work in Moonlight, so you'll need to plug in a keyboard to give this a try.

**Comment by khiven** [comment]
Hello again.

I updated Moonlight to latest version. Also reinstalled Geforece Experience just in case. Same results. Both the PC and the Link are under the same network and using ethernet.

After that I tried your suggestion, I put the PC IP on moonlight and I was able to add it and connect.

Don't really know why the auto discovery is not working, I don't have any weird restriction on my network. In any case it worked that way, just wanted to metion in case it helps anyone.
